backend/tsconfig.json

```diff
@@ -1,8 +1,13 @@
 {
+    "exclude": ["test"],
     "extends": "../configs/tsconfig.composite.json",
     "compilerOptions": {
+        "target": "es2019",
```

Comment:
That's due to `ts-node` limitations (for consumers of `cloudify-ui-common/backend`) - we should use the same target version as for ts-node (which means that e.g. Node v12 requires ES2019). 
There should be better solution here. Any suggestions?